### PR TITLE
Add new page about to the route

### PR DESCRIPTION
**Pull request by alejandrocabriales**
# Ticket

https://trello.com/c/QAVgnXw4/287-user-story-ecosystem-actor-profile-page

# What solved
- [X]  Should create a new route for the profile page with feature flags
- [X] Add scaffold for the page
- [X]  Should add a link to navigate to this page the in the actor table

# Description
This adds a scaffold for the page and adds feature flags

### CODE REVIEW

File: feature-flags/feature-flags.production.ts [hunk 1]
```diff
@@ -14,4 +14,5 @@ export const flagsProduction: FeatureFlagsInterface = {
   FEATURE_ACCOUNTS_SNAPSHOT: false,
   FEATURE_TEMPORARY_ACCOUNTS_SNAPSHOT_PAGE: false,
   FEATURE_ECOSYSTEM_ACTORS: false,
+  FEATURE_ECOSYSTEM_ACTORS_ABOUT: true,
```
Comment (YasielCabrera) [inline feature-flags/feature-flags.production.ts:17]: This page is under development so this should be set to false on production

File: feature-flags/feature-flags.interface.ts [hunk 2]
```diff
@@ -10,6 +10,7 @@ export interface FeatureFlagsInterface {
   FEATURE_TRANSPARENCY_COMMENTS: boolean; // disable Comments Tab
   FEATURE_RECOGNIZED_DELEGATES: boolean; // disable  Recognized Delegates page
   FEATURE_ACCOUNTS_SNAPSHOT: boolean; // disable Accounts Snapshot tab in the expense reports tab
-  FEATURE_ECOSYSTEM_ACTORS: boolean; // disable Accounts Snapshot page
+  FEATURE_ECOSYSTEM_ACTORS: boolean; // disable Actors Snapshot page
```
Comment (YasielCabrera) [inline feature-flags/feature-flags.interface.ts:13]: Could you update the comment as this is not a "snapshot page"

File: feature-flags/feature-flags.interface.ts [hunk 3]
```diff
@@ -10,6 +10,7 @@ export interface FeatureFlagsInterface {
...
   FEATURE_TEMPORARY_ACCOUNTS_SNAPSHOT_PAGE: boolean; // disable Accounts Snapshot page
+  FEATURE_ECOSYSTEM_ACTORS_ABOUT: boolean; // disable Actors Snapshot page
```
Comment (YasielCabrera) [inline feature-flags/feature-flags.interface.ts:15]: Could you update the comment as this is not a "snapshot page"

Comment (alejandrocabriales) [reply feature-flags/feature-flags.interface.ts]: you are right

File: feature-flags/feature-flags.interface.ts (hunk 2 above)
Comment (alejandrocabriales) [reply feature-flags/feature-flags.interface.ts]: feature/ecosystem-actor-profile-page

File: feature-flags/feature-flags.production.ts (hunk 1 above)
Comment (alejandrocabriales) [reply feature-flags/feature-flags.production.ts]: done

**Pull request merged**